modules/core/src/shaderlib/project/constants.js

```diff
@@ -0,0 +1,9 @@
+export const PROJECT_COORDINATE_SYSTEM = {
+  LNG_LAT: 1.0,
```

Comment:
Drop the underscore: `LNGLAT`

Reply:
The shader code relies on the _ being there.